LICENSE

```diff
@@ -18,4 +18,6 @@ freely, subject to the following restrictions:
 
     3. This notice may not be removed or altered from any source
     distribution.
+    
+    4. This software version is Nostalgy++, a port to Thunderbird 68
```

Comment:
I propose to keep the name "nostalgy" (and to drop this new point 4).

Reply:
@opto You have marked this conversation as resolved, but the point 4 is still there? Did you already push your changes or did you discuss this point elsewhere?